### PR TITLE
Added g15 test-case for RHMI 2.5.0 release testing

### PR DESCRIPTION
**Pull request by tonyxrmdavidson**
Added test case for Solution Explorer customer back-up time configuration

### CODE REVIEW

File: test-cases/tests/walkthroughs/g15-verify-soluton-explorer-customer-settings-backup.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - 2.5.0
+---
+
+# G15 - Verify soluton explorer customer settings backup
+
+Acceptance Criteria:
+
+When logged into solution explorer as a customer-admin
+
+1. Select ? in the top right hand corner to check the Solution Explorer version. It should be 2.26.1.
```
Comment (pawelpaszki) [inline test-cases/tests/walkthroughs/g15-verify-soluton-explorer-customer-settings-backup.md:13]: should it not be more generic, e.g. 

Select ? in the top right hand corner to check the Solution Explorer version and confirm that it matches the version obtained via CLI: `oc get rhmi rhmi -n redhat-rhmi-operator -o json | jq -r '.status.stages."solution-explorer".products."solution-explorer".version'`

Comment (tonyxrmdavidson) [reply test-cases/tests/walkthroughs/g15-verify-soluton-explorer-customer-settings-backup.md:13]: Agreed

File: test-cases/tests/walkthroughs/g15-verify-soluton-explorer-customer-settings-backup.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - 2.5.0
+---
+
+# G15 - Verify soluton explorer customer settings backup
+
+Acceptance Criteria:
+
+When logged into solution explorer as a customer-admin
+
+1. Select ? in the top right hand corner to check the Solution Explorer version and confirm that it matches the version obtained via CLI: oc get rhmi rhmi -n redhat-rhmi-operator -o json | jq -r '.status.stages."solution-explorer".products."solution-explorer".version'
```
Comment (pawelpaszki) [inline test-cases/tests/walkthroughs/g15-verify-soluton-explorer-customer-settings-backup.md:13]:
```suggestion
1. Select ? in the top right hand corner to check the Solution Explorer version and confirm that it matches the version obtained via CLI: `oc get rhmi rhmi -n redhat-rhmi-operator -o json | jq -r '.status.stages."solution-explorer".products."solution-explorer".version'`
```

**Pull request merged**